package.json

```diff
@@ -60,6 +55,8 @@
     "postcss": "^8.5.4",
     "postcss-loader": "^8.1.1",
     "postcss-preset-env": "^10.2.1",
+    "prettier": "^3.6.2",
```

Comment:
With Prettier 3.6, we can enable the `--experimental-cli` flag for a performance boost.

ref https://github.com/electron/website/pull/841